core/src/main/scala/kafka/tools/ReplicaVerificationTool.scala

```diff
@@ -143,11 +152,11 @@ object ReplicaVerificationTool extends Logging {
     }
 
     val filteredTopicMetadata = topicsMetadata.filter { topicMetaData =>
-      topicWhiteListFiler.isTopicAllowed(topicMetaData.name, excludeInternalTopics = false)
+      topicIncludeFiler.isTopicAllowed(topicMetaData.name, excludeInternalTopics = false)
```

Comment:
looks like there was a typo here before
```suggestion
      topicIncludeFilter.isTopicAllowed(topicMetaData.name, excludeInternalTopics = false)
```